### PR TITLE
chore(tools-offline): update list of project dependencies

### PR DESCRIPTION
**Pull request by Klaus-Keller**
Update file `docs/tools-offline-instructions/info/projectModules.json` which is used in scripts to make SAP Fiori tools offline available.

### CODE REVIEW

File: docs/tools-offline-instructions/info/projectModules.json [hunk 1]
```diff
@@ -81,6 +87,9 @@
         "^3.0.2",
         "3.0.2"
     ],
+    "sqlite3": [
```
Comment (longieirl) [inline docs/tools-offline-instructions/info/projectModules.json:90]: Why is this required?

Comment (Klaus-Keller) [reply docs/tools-offline-instructions/info/projectModules.json:90]: There is a script running to collect all dependencies from generated projects. Not sure if I should actively filter out anything. I PM you some details.

Comment (longieirl) [reply docs/tools-offline-instructions/info/projectModules.json:90]: Thanks for the offline explanation.

**Review by longieirl — APPROVED** [review]
Comments addressed, changes look good.
